src/services/source/nativeIntegration.ts

```diff
@@ -42,7 +42,7 @@ export default class NativeIntegrationSourceService implements IntegrationSource
         } catch (error: FixMe) {
           const metaTO = this.getTags();
           stats.increment('source_transform_errors', {
-            sourceType,
+            source: sourceType,
             version,
```

Comment:
Does `version` also need to be updated to key-value pair ?

Reply:
I dont think so

Reply:
Not necessary because the key matches.